### ISSUE TITLE
Disable Image in Header

### ISSUE THREAD

**Issue by timbrueggenthies** [jancborchardt/unsplash#20]
As of updating to the newest stable Nextcloud Release (13.0.2), i encountered a problem with the menubar at the top of the cloud UI. It seems as if the Plugin also loads an image for this menubar, resulting in a background image rather than the standart background color chosen.

Screenshot:
![unbenannt](https://user-images.githubusercontent.com/32687085/39717827-b204a3e4-5234-11e8-94d6-7382098d3b0a.JPG)

For every new Page or reload, the image also get's refreshed with a new image.

Really appreciate the plug-in!

Edit - I now saw it is officially proposed like that (my bad). Would wish for the possibility to disable the image in the header.

**Comment by agrudsky** [comment]
This is already an issue, although it hasn't been updated in a while - #14. I suppose we're waiting on progress from the dev.

**Comment by pfnegrini** [comment]
I would love as well to have the image in the header to be disabled as config option.

**Comment by jancborchardt** [comment]
This is a duplicate of #14 as @agrudsky mentioned. It will be fixed with the next release with a setting to disable the header background. :)
